src/parser/codeParser.ts

```diff
@@ -1,6 +1,6 @@
 import { parse, ParserOptions } from "@babel/parser";
 
-const testTokens = ["describe", "it", "test"];
+const testTokens = ["suite", "describe", "context", "it", "specify", "test"];
```

Comment:
It would be nice if you could add a couple unit tests for the code parser 🙃

Reply:
Sorry about that! I'll do it in another branch where I am refactoring a few parts of the code. I know it's not a good practice